portal/src/main/webapp/onco_query_lang_desc.jsp

```diff
@@ -30,26 +30,52 @@
  - along with this program.  If not, see <http://www.gnu.org/licenses/>.
```

Comment:
this file is basically `Ctrl+C` `Ctrl+V` from news.jsp, with the page layout and title changed.